ui/apps/notifications/src/components/newComponents/WelcomePage.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { tw } from '@twind/core';
+import BasicCardBox from '@akashaorg/design-system-core/lib/components/BasicCardBox';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+import Card from '@akashaorg/design-system-core/lib/components/Card';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import routes, { CUSTOMIZATION_STEP_TWO } from '../../routes';
+import { RootComponentProps } from '@akashaorg/typings/ui';
+
+interface IWelcomePageProps {
+  header: string;
+  description: string;
+  leftButtonLabel?: string;
+  rightButtonLabel: string;
+  finalStep?: boolean;
+}
+
+const WelcomePage: React.FC<RootComponentProps & IWelcomePageProps> = props => {
+  const {
+    plugins,
+    leftButtonLabel,
+    rightButtonLabel,
+    header,
+    description,
+    finalStep = false,
+  } = props;
+
+  const navigateTo = plugins['@akashaorg/app-routing']?.routing.navigateTo;
+
+  const goToNextStep = () => {
+    // navigate to step 2
+    return navigateTo?.({
+      appName: '@akashaorg/app-notifications',
+      getNavigationUrl: () => routes[CUSTOMIZATION_STEP_TWO],
+    });
+  };
+
+  const goToNotificationsPage = () => {
+    // @TODO: go to notification listing page.
```

Comment:
is this route unknown yet?

Reply:
Hi Sever. I'm working on the component for this route.